tests/unit/scripting.tcl

```diff
@@ -257,16 +257,33 @@ start_server {tags {"scripting"}} {
         run_script {return redis.pcall('wait','1','0')} 0
     } {0}
 
-    test {EVAL - Scripts can't run XREAD and XREADGROUP with BLOCK option} {
+    test {EVAL - Scripts do not block on XREAD with BLOCK option} {
         r del s
         r xgroup create s g $ MKSTREAM
-        set res [run_script {return redis.pcall('xread','STREAMS','s','$')} 1 s]
+        set res [run_script {return redis.pcall('xread','STREAMS','s','$')} 0]
```

Comment:
why did you remove the key name?

Reply:
this script doesn't use KEYS or ARGV array, so the key name is unnecessary.

Reply:
the script does use the key `s`, so at the very least we need to also pass it as an argument so the script will be sent to the right shard (when it runs in cluster mode), otherwise it can run on a different shard than the command above.
if we want, we can also fix it to use `KEYS`, but i'm also ok just keeping it as it was

Reply:
ok, let's just keep it